<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFrame</class>
 <widget class="QFrame" name="SendFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
color: rgb(0, 0, 0);
background-color: rgb(250, 250, 250);
font-family: Poppins;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="m_send_frame_main_layout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>29</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; 
font-size: 20px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#000000;&quot;&gt;Transfer PK&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_backButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="m_transfersScrollarea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>981</width>
        <height>425</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="m_send_frame_layout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>445</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="m_feeSpin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>298</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="6">
      <widget class="QDoubleSpinBox" name="m_donateSpin">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="m_paymentIdEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="donateCheckBox">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px;</string>
       </property>
       <property name="text">
        <string>Fee</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px;</string>
       </property>
       <property name="text">
        <string>Payment ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Donate 100 PK to Dev Fund</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_sendButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Send PK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_clearAllButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_addRecipientButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Add Recipient</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_generatePaymentIdButton">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
font-size: 13px;
font-family: Poppins;</string>
       </property>
       <property name="text">
        <string>Generate ID</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; 
color: rgb(170, 0, 0);
font-size:12px;
font: bold;</string>
       </property>
       <property name="text">
        <string>Account Balance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_balanceLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);
padding: 3px; 
font-size:12px;
color: rgb(0, 0, 127);
font: bold;</string>
       </property>
       <property name="text">
        <string notr="true">0.00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_tickerLabel">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_paymentIdEdit</tabstop>
  <tabstop>m_generatePaymentIdButton</tabstop>
  <tabstop>m_sendButton</tabstop>
  <tabstop>m_clearAllButton</tabstop>
  <tabstop>m_addRecipientButton</tabstop>
  <tabstop>m_transfersScrollarea</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_addRecipientButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>addRecipientClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_generatePaymentIdButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>generatePaymentIdClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_clearAllButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>clearAllClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_backButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>backClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1198</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendClicked()</slot>
  <slot>backClicked()</slot>
  <slot>clearAllClicked()</slot>
  <slot>addRecipientClicked()</slot>
 </slots>
</ui>
